neon_utils/default_configurations/default_user_conf.yml

```diff
@@ -48,3 +48,8 @@ location:
   country: United States
   tz: America/Los_Angeles
   utc: '-8.0'
+
+speed_mode:
```

Comment:
I think `speed_mode` belongs in a category like `skills` or `interface` or something more generic. I plan on moving `dialog_mode` and `use_hesitation` to user config out of local config, since these can be handled per-user.

Main reason not to use `interface` is to avoid mirroring ngi_local_conf.